build/scripts/build.ps1

```diff
@@ -16,9 +16,10 @@
 
 [CmdletBinding(PositionalBinding=$false)]
 param (
+    [string]$configuration = "Debug",
```

Comment:
Is this just reflecting the arcade model?

Reply:
Yes. I was thinking we could add `-release` as well, if that's more convenient. On the other hand PowerShell allows to shorten and use e.g. `-con Release`, so perhaps it's not necessary. I don't think IDE devs are using release builds that often. Let me know if this is common pattern for compiler devs that we should keep a short command for.

Reply:
I don't think anyone uses it enough that it's worth keeping if arcade does it differently.

Reply:
:(